### PR TITLE
40 handle no image place screen

### PR DESCRIPTION
**Pull request by darot-chen**
- Add loading shimmer
- handle null or length 0 of images
Screen Shot 2021-07-23 at 12 13 38 PM
Screen Shot 2021-07-23 at 12 13 42 PM

### CODE REVIEW

File: lib/screens/places/places_screen.dart [hunk 1]
```diff
@@ -44,38 +45,52 @@ class _PlacesScreenState extends State<PlacesScreen> with SingleTickerProviderSt
...
-                      PlaceModel place = places[index];
-                      return buildPlaceCard(place);
-                    },
-                  ),
-                ),
-              );
-            },
-          )
-        ],
+    return DefaultTabController(
```
Comment (theachoem) [inline lib/screens/places/places_screen.dart]: no need DefaultTabController if we have controller

**Comment by theachoem** [comment]
@darot-chen please add a screenshot of shimmer

**Review by theachoem — APPROVED** [review]

**Comment by darot-chen** [comment]
> @darot-chen please add a screenshot of shimmer

Don't merge yet I'll add new image model

**Review by theachoem — APPROVED** [review]
👍👍👍👍

**Pull request merged**